### PR TITLE
chore: Update ckb to v0.100

### PR DESCRIPTION
**Pull request by TheWaWaR**
### Changes:
* Rename `uncles_hash` in header to `extra_hash`
* Add `ScriptHashType::Data1` variant
* Add `OutputsValidator` to `send_transaction` rpc method

### CODE REVIEW

File: test/src/spec/plugin.rs [hunk 1]
```diff
@@ -60,7 +60,11 @@ impl Spec for Plugin {
 
         let output = setup.cli(&format!("wallet transfer --from-account {} --to-address ckt1qyqt8xaupvm8837nv3gtc9x0ekkj64vud3jq5t63cs --capacity 1000 --tx-fee 0.1", Miner::address()));
         // Means the signature is filled but is wrong: https://github.com/nervosnetwork/ckb-system-scripts/wiki/Error-codes
```
Comment (doitian) [inline test/src/spec/plugin.rs:62]: Minor: outdated comment

**Review by doitian — COMMENTED** [review]
Minor suggestions

File: test/src/spec/plugin.rs (hunk 1 above)
Comment (TheWaWaR) [reply test/src/spec/plugin.rs:62]: The link or the description is outdated comment?

Is https://nervosnetwork.github.io/ckb-script-error-codes/by-data-hash/709f3fda12f561cfacf92273c57a98fede188a3f1a59b1f888d113f9cce08649.html#-31 the right link?

Comment (doitian) [reply test/src/spec/plugin.rs:62]: Yes

**Review by doitian — APPROVED** [review]

**Review by yangby-cryptape — APPROVED** [review]

**Pull request merged**